README.md

```diff
@@ -4,32 +4,32 @@ A collection of code snippets mostly designed to be dropped into the data harves
 
 ## Usage
 
-- Familiarize yourself with the [harvesting instructions](https://github.com/datarefugephilly/workflow/tree/master/harvesting-toolkit) in the Data Rescue workflow repo.  Within the [pipeline app](http://harvest-pipeline.herokuapp.com/), click `Download Zip Starter` from the page related to a URL that you have checked out. 
+- Familiarize yourself with the [harvesting instructions](https://github.com/datarefugephilly/workflow/tree/master/harvesting-toolkit) in the Data Rescue workflow repo.  Within the [pipeline app](http://harvest-pipeline.herokuapp.com/), click `Download Zip Starter` from the page related to a URL that you have checked out.
 - unzip the zipfile
 - Choose a tool that seems likely to be helpful in capturing this particular resource, and copy the contents of its directory in this repo to the `tools` directory, e.g. with:
   ```
   cp -r harvesting-tools/TOOLNAME/* RESOURCEUUID/tools/
   ```
 - Adjust the base URL for the resource along with any other relevant variables, and tweak the content of the tool as necessary
-- After the resource has been harvested, proceed with the further steps in the [workflow](https://github.com/datarefugephilly/workflow/). 
+- After the resource has been harvested, proceed with the further steps in the [workflow](https://github.com/datarefugephilly/workflow/).
 
 ## Matching Tools and Datasets
 
-Each tool in this repo has a fairly specific use case. Your choice will depend on the shape and size of the data you're dealing with. Some datasetswill require more creativity/more elaborate tools. If you write a new tool, please [add it to the repo](#contributing). 
+Each tool in this repo has a fairly specific use case. Your choice will depend on the shape and size of the data you're dealing with. Some datasetswill require more creativity/more elaborate tools. If you write a new tool, please [add it to the repo](#contributing).
```

Comment:
I think this link might need to be changed to `./CONTRIBUTING.md`? probably a relic of the previous iteration